sw/device/lib/testing/test_framework/BUILD

```diff
@@ -140,6 +143,8 @@ cc_library(
         "//sw/device/lib/runtime:hart",
         "//sw/device/lib/runtime:ibex",
         "//sw/device/lib/runtime:log",
+        "//sw/device/silicon_creator/lib/base:static_critical_boot_measurements",
```

Comment:
nit: Can you update the comment above `ottf_start` to include details on how the target is used in E2E tests?